### DISCUSSION TITLE
Slight bit of convenience regarding the project tree

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1907] [Ideas]
I'm wondering, why is the icon which tells whether an item is "active" or "inactive" (checkmark or cross) in the project tree not responsive to double clicks to change its state? To me this would seem a tiny bit of convenience which would go a long way. Granted, not used that often, but it's a small thing.

Yes, I'm aware there's the popup menu option. However, that seems like more work if you want to change the setting of a bunch of items.

Or is it already included and is my desktop environment (Cinnamon on Linux Mint) misbehaving?

EDIT: I have noticed when I double click an item in the project tree, the state icon changes color ever so slightly in the selection.

**Comment by vkbo** [comment]
It's not a checkbox, just an icon resembling one.

**Reply by xahodo** [reply]
Still, it would be convenient to be able to change it using a double click.

**Reply by vkbo** [reply]
Double-click is already in use to open the document. If a small region of the line suddenly changed behaviour, people may end up toggling this flag without noticing it. I don't think the change gives enough benefit for causing this change in behaviour. Also, it's not standard UI behaviour, so people wouldn't expect it.

Not sure why it would be needed either. It's not a setting you should need to change often.

**Comment by vkbo** [comment]
> EDIT: I have noticed when I double click an item in the project tree, the state icon changes color ever so slightly in the selection.

That's just the overlay from the selection highlighting. The icon itself doesn't change.
